Tiqr Client.xcodeproj/project.pbxproj

```diff
@@ -464,9 +464,9 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				MARKETING_VERSION = 3.0.11;
+				MARKETING_VERSION = 3.1;
 				PRODUCT_BUNDLE_IDENTIFIER = org.tiqr.surfnet;
-				PRODUCT_NAME = "$(TARGET_NAME)";
+				PRODUCT_NAME = TiqrClient;
```

Comment:
Is it wise to hard code the PRODUCT NAME?

Reply:
The problem was that there were spaces in certain target names. This is the name of the executable, and not visible to the user.

Reply:
so why not rename the target?